examples/system_tray.rs

```diff
@@ -30,6 +33,13 @@ fn main() {
   // open new window menu item
   let open_new_window_element = submenu.add_item(MenuItemAttributes::new("Open new window"));
 
+  #[cfg(target_os = "macos")]
+  let mut is_accessory = false;
+  // Accessory activation policy toggle
+  #[cfg(target_os = "macos")]
+  let mut activation_policy_accessory =
+    submenu.add_item(MenuItemAttributes::new("Accessory Activation Policy").with_selected(false));
```

Comment:
No need to add an example of it since it is platform-specific.

Reply:
Removed it